### DISCUSSION TITLE
The interface is broken in the latest version of the program

### DISCUSSION THREAD

**Discussion by youngishcord** [inviwo/inviwo#1606] [Q&A] [answered]
Hello
I have build the latest version of invivo several times, and each time I have a problem with the layout, as shown in the picture. I do not know what could be causing this problem, but it really gets in the way. When build on another PC, the problem did not appear.

![xCrKUmMAcMo](https://github.com/user-attachments/assets/964a90af-2590-48d4-804c-06bc0d8ac75d)

Can you please tell me how to fix this, or how to identify the source of the problem in order to create a full-fledged problem?

**Comment by petersteneteg** [comment, marked as answer]
Hi

This is a known issue we have observed with the state management in Qt.  Unfortunately it is hard to debug since we have not figured out how to reproduce it.

As a workaround you can delete all the contents of `Computer\Dator\HKEY_CURRENT_USER\Software\Inviwo Foundation`
in the windows registry and the state should revert back to the default.

**Reply by youngishcord** [reply]
Yes, it really works. Thank you very much!

**Reply by martinfalk** [reply]
> Yes, it really works. Thank you very much!

Alternatively, you can also start Inviwo with the command line argument `--reset-window-state`.
